inst/dust/cows.cpp

```diff
@@ -137,7 +150,20 @@ class cows {
         E_next[j] = E[j] + n_SE - n_EI;
         I_next[j] = I[j] + n_EI - n_IR;
         R_next[j] = R[j] + n_IR;
+
+        // Check if we have declared an outbreak in this herd, add
+        // that to the region total if so.
+        if (outbreak[j]) {
```

Comment:
When would outbreak[j] be set as true? I can't work out when that would happen. Should it not happen somewhere in the else{} part?

outbreak[j] is tracking whether or not herd j has ever had an outbreak, I believe?

Reply:
No I see it now, dust/mcstate knows that a parameter marked _next will be the starting point for the next timestep.